[docs][migrate] changes to yb-voyager cli reference page (#15738)
yugabyte/yugabyte-db: docs/content/preview/migrate/reference/yb-voyager-cli.md

yb-voyager export schema [ <arguments> ... ]
```

The valid *arguments* for export schema are described in the following table:

| Argument | Description/valid options |
| :------- | :------------------------ |
| [-e, --export-dir](#export-dir) <path> | Path to the directory containing the data files to export. |
| [-h, --help](#command-line-help) | Command line help. |

> | [--oracle-db-sid](#oracle-db-sid) <SID> | Oracle System Identifier. |

Author: @amit-yb , @sanyamsinghal : Do you want us to call out or we can assume that it applies only to Oracle?

> | [--oracle-db-sid](#oracle-db-sid) <SID> | Oracle System Identifier. |

Reviewer: My point was partly prompted by the fact that there are 3 oracle arguments, but only one is tagged as Oracle only. Add that to the other 2 would be fine.

        --target-db-user username \
        --target-db-password password \ # Enclose the password in single quotes if it contains special characters.
        --target-db-name dbname \
        --target-db-schema schemaName # MySQL and Oracle only
```

### import data

[Import the data](../migrate-steps/#import-data) to the target YugabyteDB.

#### Syntax

```sh
yb-voyager import data [ <arguments> ... ]
```

The valid *arguments* for import data are described in the following table:

| Argument | Description/valid options |
| :------- | :------------------------ |
| [--batch-size](#batch-size) <number> | Size of batches generated for ingestion during [import data]. |
| --continue-on-error |

> | [--disable-pb](#disable-pb) | Hides progress bars. |

Reviewer: ```suggestion
| [--disable-pb](#disable-pb) | Hide progress bars. |
```

Specifies the path to the directory containing the data files to export.

An export directory is a workspace used by yb-voyager to store the following:

- exported schema DDL files
- export data files
- migration state
- log file

> ### --source-db-type

Reviewer: Why are the arguments in the Arguments section in this order? Seems random.

Author: I don't think order matters here because we are anyway linking them for different commands. In any case , will keep it as an improvement task.

Specifies the schema of the source database. Not applicable for MySQL.

For Oracle, you can specify only one schema name using this option.

> For PostgreSQL, you can specify a list of comma separated schema names.

Reviewer: ```suggestion
For PostgreSQL, you can specify a list of comma-separated schema names.
```

> Case sensitive schema names are not yet supported.

Reviewer: known issue? link?

Specifies the port number of the machine on which the target database server is running.

Default: 5433

### --target-db-user

> Specifies the username of the target database.

Reviewer: ```suggestion
Specifies the username in the target database to be used for the migration.
```

> Specifies the password of the target database.

Reviewer: ```suggestion
Specifies the password for the the target database user to be used for the migration.
```

If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password.

> If the password contains special characters that are interpreted by the shell (for example, # and $), enclose it in single quotes.

Reviewer: ```suggestion
If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes.
```

### --target-db-name

Specifies the name of the target database.

Default: yugabyte

### --target-db-schema

> Specifies the schema of the target database. Applicable only for MySQL and Oracle.

Reviewer: ```suggestion
Specifies the schema of the target database. MySQL and Oracle migrations only.
```

### --parallel-jobs

Specifies the number of parallel COPY commands issued to the target database.

> If yb-voyager can determine the total number of cores `N` in the target YugabyteDB cluster, it will use `N/2` as the default value of `--parallel-jobs`. Otherwise, it will default to twice the number of nodes in the cluster.

Reviewer: ```suggestion
Default: If yb-voyager can determine the total number of cores `N` in the target YugabyteDB cluster, it uses `N/2` as the default. Otherwise, it defaults to twice the number of nodes in the cluster.
```

> Depending on the target YugabyteDB configuration, the value of `--parallel-jobs` should be tweaked such that *at most* 50% of target cores are utilised.

Reviewer: move this to before previous paragraph

### --delimiter

> Default: comma(,); can be changed to '\t' (tab), pipe(|), or any other character.

Reviewer: ```suggestion
Default: comma (,); can be changed to '\t' (tab), pipe(|), or any other character.
```

### --use-orafce

> Enable using orafce extension in export schema. Applicable for Oracle only.

Reviewer: could use a little more info here - what's it used for, where does the extension need to be installed, the source and the target? just the source? what if you turn this flag on but haven't installed the extension?

'Oracle migrations only.'

Author: @sanyamsinghal: Could you please provide some details about this ?

Comma-separated list of tables to exclude while exporting data.

### --send-diagnostics

Controls whether to send diagnostics information to Yugabyte.

Default: true

### --verbose

Displays extra information in the output.

Default: false

### --disable-pb

> Specify this argument to not show progress bars.

Reviewer: ```suggestion
Do not display progress bars.
```

Default: false

---

> ## SSL Connectivity

Author: cc: @amit-yb , @sanyamsinghal
